math/CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Ref: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.j
 ### Bug Fixes
 
 * [#15714](https://github.com/cosmos/cosmos-sdk/pull/15714) `FormatInt` returns an error on empty string.
+* [#16266](https://github.com/cosmos/cosmos-sdk/pull/16266) fix: legacy dec power mut zero exponent precision.
```

Comment:
It should be under a bug fixes section under [Unreleased] (sorry to be nitpicky lol)